turbo.json

```diff
@@ -36,6 +36,7 @@
     },
     "api-check": {
       "dependsOn": ["^build"]
-    }
+    },
+    "api-check:update": {}
```

Comment:
I think this is missing 
```
"api-check:update": {
     "outputs": ["temp/*", "ect/*"]
}
```

Reply:
what is the outputs section used for? 🤔

Reply:
I think that's only useful if some other task would depend on `update`, right? 
For that case I don't really see a scenario where we would want to keep that info cached, as a call to it means something went wrong/API changed